src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -9,13 +9,21 @@ import TouchEventManager from './TouchEventManager';
 import { State } from '../../State';
 import { isPointerInBounds } from '../utils';
 import EventManager from './EventManager';
+import { Config, MouseButton } from '../interfaces';
 
 export class GestureHandlerWebDelegate
   implements GestureHandlerDelegate<HTMLElement>
 {
   private view!: HTMLElement;
   private gestureHandler!: GestureHandler;
   private eventManagers: EventManager<unknown>[] = [];
+  private boundDisableContextMenu: (e: MouseEvent) => void;
+  private boundEnableContextMenu: (e: MouseEvent) => void;
+
+  constructor() {
+    this.boundDisableContextMenu = this.disableContextMenu.bind(this);
+    this.boundEnableContextMenu = this.enableContextMenu.bind(this);
```

Comment:
Without those bindings you'll get this error:
``` 
error  Avoid referencing unbound methods which may cause unintentional scoping of `this`.
If your function does not access `this`, you can annotate it with `this: void`, or consider using an arrow function instead  @typescript-eslint/unbound-method
```

But looking at it I assume it is better to use `this:void` instead of binding `this`.

Changed in [c75fdb7](https://github.com/software-mansion/react-native-gesture-handler/pull/2676/commits/c75fdb766e6993f48b6a6511cb250e977928b6a2)